tests/unit/torch/outputs/test_classification.py

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import torch
+from torch import nn
+from torchmetrics import AUROC, Accuracy, Precision, Recall
+
+import merlin.dtypes as md
+import merlin.models.torch as mm
+from merlin.models.torch.utils import module_utils
+from merlin.schema import ColumnSchema, Schema
+
+
+class TestBinaryOutput:
```

Comment:
Would be great to add at least 2 extra tests:
1. Validate loss-calculation (after a forward-pass) 
2. Validate metric-calculation (after a forward-pass)

Reply:
I added a couple of tests that check the loss and the metrics, but it looks to me like the actual calculation would be better addressed after we implement `calculate_loss`.